interface/patient_file/summary/immunizations.php

```diff
@@ -310,12 +310,12 @@ function saveImmunizationObservationResults($id,$immunizationdata)
 <!-- supporting javascript code -->
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-1/index.js"></script>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
-
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-ui-min/index.js"></script>
```

Comment:
Actually, above is wrong.
Note that jquery version 1.2.1 is already brought in above.
Do you need to bring in version 1.9.1; if so, then should remove the line for 1.2.1 above.

Also, for the jquery-ui import, should be using:
``````
<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-ui-1-10-4/ui/jquery-ui.js"></script>